src/pages/MainPage/HeroSection.jsx

```diff
@@ -1,18 +1,22 @@
 import SearchPannel from '../../components/common/SearchPannel/SearchPannel';
```

Comment:
typo `Pannel` => `Panel`

Reply:
You can install spellchecker plugin to avoid incorrect spelling

Reply:
סליחה, אנגלית היא לא שפת האם שלי.